src/Workspaces/Core/Portable/CodeStyle/CodeStyleHelpers.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Microsoft.CodeAnalysis.CodeStyle
+{
+    internal static class CodeStyleHelpers
+    {
+        public static CodeStyleOption<bool> ParseEditorConfigCodeStyleOption(string arg)
+        {
+            var args = arg.Split(':');
+            if (args.Length != 2)
+            {
+                return CodeStyleOption<bool>.Default;
+            }
+
+            bool isEnabled = false;
+            if (!bool.TryParse(args[0], out isEnabled))
+            {
+                return CodeStyleOption<bool>.Default;
+            }
+
+            switch (args[1].Trim())
+            {
+                case "none": return new CodeStyleOption<bool>(value: isEnabled, notification: NotificationOption.None);
```

Comment:
For code styles, did we ever discuss having it be more explicit to say severity=none? My worry is an option that is "false, none" isn't exactly obvious in context what that means for somebody first seeing it.